applications/Chat/examples/README.md

```diff
@@ -154,7 +154,7 @@ torchrun --standalone --nproc_per_node=4 train_prompts.py \
          --rm_path /your/rm/model/path
 ```
 
-Prompt dataset: the instruction dataset mentioned in the above figure which includes the instructions, e.g. you can use [seed_prompts_ch.jsonl](https://github.com/XueFuzhao/InstructionWild/blob/main/data/seed_prompts_ch.jsonl) or [seed_prompts_en.jsonl](https://github.com/XueFuzhao/InstructionWild/blob/main/data/seed_prompts_en.jsonl) in InstructionWild.
+Prompt dataset: the instruction dataset mentioned in the above figure which includes the instructions, e.g. you can use the example jsonl files [here](https://github.com/hpcaitech/ColossalAI/tree/main/applications/Chat/examples/data) with our data format. (source from seed jsonl files in [InstructionWild](https://github.com/XueFuzhao/InstructionWild/blob/main/data))  
```

Comment:
It's very possible that users also downloaded the original dataset, which is confusing to them as the datasets have the same name and very similar content.